src/libs/replaceSourceOrigin.js

```diff
@@ -0,0 +1,24 @@
+import _ from 'underscore';
+import Config from '../CONFIG';
+
+/**
+ * Update the URL so images/files can be accessed depending on the config environment
+ *
+ * @param {String} urlString
+ * @returns {*|String}
+ */
+export default function replaceSourceOrigin(urlString) {
```

Comment:
To be honest I think I'd rather see all of these changes in a different PR... but if this makes it impossible for you to test, maybe we can keep them here but we should mention you're trying to also fix https://github.com/Expensify/App/issues/14227 right?

Reply:
> To be honest I think I'd rather see all of these changes in a different PR... but if this makes it impossible for you to test, maybe we can keep them here but we should mention you're trying to also fix #14227 right?

I'm not exactly sure how this issue will be resolved
- #14227 

I might drop the changes or submit a separate PR if it turns out to be the accepted solution
In the meantime I keep the fix here to avoid reporting it as a problem of (and the react-native-web PR)
- #12603

Reply:
It won't make the PR impossible to test, but people will get CORS errors (on DEV) for images that were uploaded on staging